blueocean-rest/src/main/java/io/jenkins/blueocean/rest/factory/BlueArtifactFactory.java

```diff
@@ -0,0 +1,37 @@
+package io.jenkins.blueocean.rest.factory;
```

Comment:
@abishek3876 this is the package that contains our primary extension points. We are also working on a way for developers to extend the user interface directly but it is immature at this time. If there are specific places in the UI that you think could or should be extensible, please open a ticket.